<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompraMenu</class>
 <widget class="QWidget" name="CompraMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SELECCIONAR DOCUMENTO</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <item>
      <property name="text">
       <string>Orden</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Factura</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Boleta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Guía de Remisión Remitente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flete</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nota de Crédito</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nota de Débito</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Saldo</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_salir">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_salir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
